bitnami/sealed-secrets/values.yaml

```diff
@@ -477,6 +516,18 @@ rbac:
   ## @param rbac.pspEnabled PodSecurityPolicy
   ##
   pspEnabled: false
+  ## @param rbac.clusterRole Specifies whether the Cluster Role resource should be created
+  ##
+  clusterRole: true
+  ## @param rbac.clusterRoleName Specifies the name for the Cluster Role resource
+  ##
+  clusterRoleName: ""
+  ## @param rbac.namespacedRoles Specifies whether the namespaced Roles should be created (in each of the specified additionalNamespaces)
+  ##
+  namespacedRoles: false
```

Comment:
Just a recommendation, should we detail here that using `clusterRole=true` and `namespacedRoles=true` will result in no clusterrole nor roles being created? I understand the logic but maybe users can be confused if they try using both.